src/org/labkey/test/pages/admin/ImportFolderPage.java

```diff
@@ -48,7 +48,27 @@ public static ImportFolderPage beginAt(WebDriverWrapper driver)
     public static ImportFolderPage beginAt(WebDriverWrapper driver, String containerPath)
     {
         driver.beginAt(WebTestHelper.buildURL("admin", containerPath, "importFolder", Maps.of("tabId", "import")));
-        return new ImportFolderPage(driver.getDriver());
+        ImportFolderPage page = new ImportFolderPage(driver.getDriver());
+        page.setDriver(driver.getDriver());
```

Comment:
Calling `setDriver` here should be unnecessary. There's a problem with the constructor if this is required.

Reply:
Yes, there's a problem- and I couldn't figure it out.
Note that ImportFolderPage overrides getWrappedDriver() and throws IllegalStateException when _driver is null?

There's something screwy about how this class implements FolderManagementTab, local _driver is consistently null (base class is where driver lives, right?)

Reply:
I didn't want to pull that thread and end up with a whole unraveled sweater

Reply:
I must've done something wrong in that base class. We can figure out what's up with that some other time.